examples/generative/Main.hs

```diff
@@ -95,19 +97,20 @@ handleEvent wenv node model evt = case evt of
   GenerativeInit -> [setFocusOnKey wenv "activeType"]
 
 main :: IO ()
-main = do
-  startApp model handleEvent buildUI config
+main = config >>= startApp model handleEvent buildUI
   where
     model = GenerativeModel CirclesGrid False def def
-    config = [
-      appWindowTitle "Generative",
-      appTheme darkTheme,
-      appFontDef "Regular" "./assets/fonts/Roboto-Regular.ttf",
-      appInitEvent GenerativeInit
-      ]
+    config = do
+      appFontDef'Regular <- (appFontDef "Regular" . pack) <$> getDataFileName "assets/fonts/Roboto-Regular.ttf"
```

Comment:
we want to read from the distributed path instead of the source project repository, which might not be accessible

Reply:
reverted